TheJumpCloud/jcapi-powershell
# check draft w/ contexts

## Issues
* [SA-3119](https://jumpcloud.atlassian.net/browse/SA-3119) - Update GH Contexts for Draft Release step

## What does this solve?

Update the context in our config.yaml to allow for drafting github releases during merge to main branch. 

## Is there anything particularly tricky?

## How should this be tested?

It works, [see this run ](https://app.circleci.com/pipelines/github/TheJumpCloud/jcapi-powershell/1651/workflows/298daf24-d146-4552-aebd-1281fe4c0c38/jobs/4255)

## Screenshots

[SA-3119]: https://jumpcloud.atlassian.net/browse/SA-3119?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
